R/ForecastRegrTask.R

```diff
@@ -0,0 +1,58 @@
+#' @title Create a task for univariate forecasting
+#'
+#' @rdname Task
+#'
+#' @description Creates a task for univariate forecasting learners
+#'
+#' @export
+makeForecastRegrTask = function(id = deparse(substitute(data)), data, target,
+  weights = NULL, blocking = NULL, frequency = 1L, fixup.data = "warn",
+  check.data = TRUE) {
+  assertString(id)
+  assertClass(data, "xts")
```

Comment:
P.S. reordering the data could be a problem if the user expects the data to be in the given order, e.g. when doing `subset = ...` on later `train` calls.

Maybe it would be better to forget about `order()` and just throw an error if dates are not ascending.